src/rollup-plugin-deno-resolver/loadUrl.ts

```diff
@@ -22,9 +23,16 @@ export async function loadUrl(
     }
     case "http:":
     case "https:": {
-      const res = await fetch(url.href, fetchOpts);
+      try {
+        // get from deno cache storage
+        const file = await Cache.cache(url.href);
 
-      return await res.text();
+        return await Deno.readTextFile(file.path);
```

Comment:
Can something here throw an error? Does the cache mechanism not update the cache ( i.e. perform a fetch or equivalent ) behind the scene?

Just curious why would need the fallback to `fetch()` below.

Reply:
Also wonder how this plays with https://rollupjs.org/guide/en/#cache - if using Rollup's caching mechanism, do you know if this will add value? Regardless feels like probably a nice thing to have.

Reply:
Also wondering if this module caters for things on the filesystem, data urls etc.? If so this entire block code be simplified as:

1. Retrieve from cache ( regardless of import type )
2. Return plaintext of cached file

Reply:
> Can something here throw an error? Does the cache mechanism not update the cache ( i.e. perform a fetch or equivalent ) behind the scene?
> 
> Just curious why would need the fallback to `fetch()` below.

when a dependency is not in the deno cache, Cache it throws an error, that's why the try catch block

Reply:
> Also wonder how this plays with https://rollupjs.org/guide/en/#cache - if using Rollup's caching mechanism, do you know if this will add value? Regardless feels like probably a nice thing to have.

I'm not sure if it would add a considerable improvement, but taking from the cache I couldn't notice a significant improvement in the initial and subsequent bundler time